app/views/galleries/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<form action="/galleries/<%= @gallery.id %>" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="hidden" name="_method" value="PATCH" />
+
+  <p>Update Gallery Information</p>
+  <label for="gname">Gallery Name:</label>
+  <input type='text' name='gallery[name]' value="<%= @gallery.name %>"><br><br>
+
+  <label for="gentrycost">Entry Cost (in USD):</label>
+  <input type='text' name='gallery[entry_cost]' value="<%= @gallery.entry_cost %>"><br><br>
```

Comment:
As mentioned for /galleries/new it would be nice if this was a number field instead of a text field.